### ISSUE TITLE
exFAT Partition not detected in opencore

### ISSUE THREAD

**Issue by Yeahboi12356** [corpnewt/UnPlugged#3]
I had a problem when using this offline installation when I wanted to access the exFat partition I couldn't find the folder that I had put on the exFat partition so I couldn't access it.
And I also can't download my mac os to ssd because it is not detected by the system
Is there any way to fix this?

![IMG20240702180807](https://github.com/corpnewt/UnPlugged/assets/89899637/b056f092-5b13-409b-958b-e119c1812e84)

And also I have used the "diskutil list" tool and my exFat partition is not detected

**Comment by corpnewt** [comment]
Are you installing Sonoma?  If so - did you read through the Sonoma Notes portion of the readme?

-CorpNewt

**Comment by Yeahboi12356** [comment]
> Are you installing Sonoma? If so - did you read through the Sonoma Notes portion of the readme?
> 
> -CorpNewt

My friend send me he's efi it should normal now i hope that work

**Comment by corpnewt** [comment]
Well, as this does not appear to be an issue with the script - I'm going to close this issue.  If the script appears to be at fault, feel free to reply and I'll reopen as needed.

-CorpNewt

**Comment by abdullahcfix** [comment]
Hi, I'm trying to install Sonoma on my laptop and am having the same issue. I used gibMacOS and UnPlugged to create an installer drive and even followed the steps specific to Sonoma by having recovery files for Ventura on the EFI partition to be able to see the exFAT volume, but the recovery environment cannot see the exFAT partition on the same USB that contains Sonoma's BaseSystem.dmg, InstallAssistant.pkg, and UnPlugged.command, so I can't proceed with the installation. I've tried changing that volume to be HFS+ and APFS, tried different ports, and an external hard drive with the files on there separately, but it still doesn't see it. It doesn't even see my external NVMe to USB-C drive preformatted to APFS or HFS+ via my other hackintosh, so how can I get past this point?
